pkg/envtest/multiplecrds.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: firsts.first.example.com
```

Comment:
`firsts.first.example.com` and `seconds.second.example.com` looks very weird.
I'd suggest `captains.crew.example.com` and `firstmates.crew.example.com`. These are something we use a lot in kubebuilder testing.

Reply:
Updated PR